glossary.md

```diff
@@ -6,28 +6,97 @@
 * `Library`: The collection of all `responses`.
 * `Registry`: The collection of available `survey objects` for use in `responses`.
 * `Analysis`: The process of converting many survey responses to a Sankey diagram by an
-  `Analyst` interacting with the `Library`. Surveys are searched by title and `tags`.
+  `Analyst` interacting with the `Library`. Surveys are searched by title, `tags`, and/or `object metadata`.
 
 
-## Response Object Registry
+## Registered Survey Object Registry
```

Comment:
> Is there a difference between response objects and survey objects?

I would prefer to use the term  "Response" here, since these will be chosen during the response process, not the survey creation process. Would also prefer to avoid using the term "register" twice!

```suggestion
## Response Object Registry
```

Reply:
Sounds good